.github/workflows/plugin-actions.yml

```diff
@@ -1,10 +1,6 @@
-name: Dart
+name: Plugin actions
```

Comment:
How about using a more specific name which is common for both jobs like "Format" or "Analyze"?

Reply:
I've updated this PR as you mentioned